rir/src/compiler/translations/pir_2_rir.cpp

```diff
@@ -621,12 +622,13 @@ class Pir2Rir {
 size_t Pir2Rir::compileCode(Context& ctx, Code* code) {
     toCSSA(code);
 
-    if (compiler.shouldPrintCSSA())
+    if (compiler.debug.includes(DebugFlag::PrintLiveness))
```

Comment:
Maybe it's useful to alias `PrintFinalPir` or something? It's not immediately obvious that `PrintLiveness` gives you the final PIR bytecode (is this even correct?) before it's compiled back to RIR, and I think that's something useful to see.

Sure, there's some redundancy, but changing the condition to `compiler.debug.includes(DebugFlag::PrintLiveness | DebugFlag::PrintFinalPir)` is a small price for convenience.